### PR TITLE
data: projects added by Abhinav kumar

### PR DESCRIPTION
**Pull request by Abhinavcode13**
Added a new project on machine learning.

## Related Issue

closes the issue #1061 

## Description

This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE

### CODE REVIEW

File: src/DB/projects.json [hunk 1]
```diff
@@ -2202,5 +2202,23 @@
...
+      "YouTube": "",
+      "Twitter": "https://twitter.com/Abhinavxkumar13"
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
+        "title": "Game of thrones personality matcher",
+        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
+        "tech": ["python","Machine learning","Jupyter Notebook"]
+      }
```
Comment (priyankarpal) [inline src/DB/projects.json:2221]: you have added code previously you don't need to add full code, just add the sample code to the existing code.

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json [hunk 2]
```diff
@@ -2202,5 +2202,23 @@
         "tech": ["python","Machine learning"]
       }
```
Comment (priyankarpal) [inline src/DB/projects.json:2203]: just paste the sample code to here 
```
{
        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
        "title": "Game of thrones personality matcher",
        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
        "tech": ["python","Machine learning","Jupyter Notebook"]
      }
```

```suggestion
      },
      {
        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
        "title": "Game of thrones personality matcher",
        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
        "tech": ["python","Machine learning","Jupyter Notebook"]
      }
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: src/DB/projects.json [hunk 3]
```diff
@@ -2200,6 +2200,30 @@
...
+    },
+    "Projects": [
+      {
+        "link": "https://github.com/Abhinavcode13/Game-Of-Thrones-Personality-Matcher",
+        "title": "Game of thrones personality matcher",
+        "description": "This is a project based on data science and machine learning which used TSNE algorithm and the main aim of this project is to match the personality of the game of thrones characters and this app is hosted on streamlit framework in pycharm IDE",
+        "tech": ["python","Machine learning","Jupyter Notebook"]
       }
     ]
   },
```
Comment (priyankarpal) [inline src/DB/projects.json:2229]: remove the code

**Review by Abhinavcode13 — COMMENTED** [review]
removed the code

**Review by priyankarpal — APPROVED** [review]
Looks Good 🎉

**Comment by Abhinavcode13** [comment]
It shows type error @priyankarpal

**Comment by priyankarpal** [comment]
> It shows type error @priyankarpal

ok let me check

**Pull request merged**